docs/source/terminology.rst

```diff
@@ -2,49 +2,100 @@
+*Hindcast Ensemble*: ``m`` ensemble members are initialized from a reference simulation
+(generally a reconstruction from reanalysis) at ``n`` initialization dates and
+integrated for ``l`` lead years [Boer2016]_
+(:py:class:`~climpred.classes.HindcastEnsemble`).
+
+*Perfect Model Experiment*: ``m`` ensemble members are initialized from a control
+simulation at ``n`` randomly chosen initialization dates and integrated for ``l``
+lead years [Griffies1997]_ (:py:class:`~climpred.classes.PerfectModelEnsemble`).
+
+*Reconstruction/Assimilation*: A "reconstruction" is a model solution that uses
+observations in some capacity to approximate historical conditions. This could be done
+via a forced simulation, such as an OMIP run that uses a dynamical ocean/sea ice core
+with reanalysis forcing from atmospheric winds. This could also be a fully data
+assimilative model, which assimilates observations into the model solution.
+
+*Uninitialized Ensemble*: In this framework, an *uninitialized ensemble* is one that
+is generated by perturbing initial conditions only at one point in the historical run.
+These are generated via micro (round-off error perturbations) or macro (starting from
+completely different restart files) methods. Uninitialized ensembles are used to
+approximate the magnitude of internal climate variability and to confidently extract
+the forced response (ensemble mean) in the climate system. In ``climpred``, we use
+uninitialized ensembles as a baseline for how important (reoccurring) initializations
+are for lending predictability to the system. Some modeling centers (such as NCAR)
+provide a dynamical uninitialized ensemble (the CESM Large Ensemble) along with their
+initialized prediction system (the CESM Decadal Prediction Large Ensemble). If this
+isn't available, one can approximate the unintiailized response by bootstrapping a
+control simulation.
+
+Forecast Assessment
+###################
+
+*Accuracy*: The average degree of correspondence between individual pairs of forecasts
+and observations [Murphy1988]_; [Jolliffe2011]_. Examples include Mean Absolute Error
+(MAE) and Mean Square Error (MSE). See `metrics <metrics.html>`_.
+
+*Association*: The overall strength of the relationship between individual pairs of
+forecasts and observations [Jolliffe2011]_. The primary measure of association is the
+Anomaly Correlation Coefficient (ACC), which can be measured using the Pearson
+product-moment correlation or Spearman's Rank correlation. See
+`metrics <metrics.html>`_.
+
+*(Potential) Predictability*: This characterizes the "ability to be predicted"
+rather than the current "ability to predict." One acquires this by computing a metric
+(like the anomaly correlation coefficient (ACC)) between the prediction ensemble and a
```

Comment:
(Potential) Predictability is independent of metric, its  determined by the experiment setup, eg. what the comparison is against.